### PR TITLE
[bugfix] Center incoming call toast button icons

### PR DESCRIPTION
**Pull request by edwardlee-msft**
# What
Center incomingCallAlerts default button icons

image

# Why
https://skype.visualstudio.com/SPOOL/_workitems/edit/2985959
https://skype.visualstudio.com/SPOOL/_workitems/edit/3009081

# How Tested
Storybook Incoming Call Toast example.
Tested through 1:N call composite sample in story book as well.

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/storybook/stories/Examples/IncomingCallAlerts/components/IncomingCallAlerts.tsx [hunk 1]
```diff
@@ -57,12 +57,12 @@ export const IncomingCallToast = (props: IncomingCallToastProps): JSX.Element =>
...
           className={incomingCallRejectButtonStyle}
           onClick={() => onClickReject()}
-          onRenderIcon={() => <CallEnd20Filled />}
+          onRenderIcon={() => <CallEnd20Filled style={{ height: '1.25rem' }} />}
         />
         <DefaultButton
           className={incomingCallAcceptButtonStyle}
           onClick={() => onClickAccept()}
-          onRenderIcon={() => <Call20Filled />}
+          onRenderIcon={() => <Call20Filled style={{ height: '1.25rem' }} />}
```
Comment (mgamis-msft) [inline packages/storybook/stories/Examples/IncomingCallAlerts/components/IncomingCallAlerts.tsx:65]: Looks good to me. Just wondering if this change in icon was all it took to center it?

**Comment by mgamis-msft** [comment]
Took a look at the chromatic and the incoming call toast icons look centered :) Although I did notice the the incoming call modal's icon not centered.
image
Is there another bug for this too?

**Review by mgamis-msft — APPROVED** [review]

File: packages/storybook/stories/Examples/IncomingCallAlerts/components/IncomingCallAlerts.tsx (hunk 1 above)
Comment (edwardlee-msft) [reply packages/storybook/stories/Examples/IncomingCallAlerts/components/IncomingCallAlerts.tsx]: That's right, 
I actually changed it to use "display: 'flex'" as it is less obtrusive if Contoso ever decided to change height dimensions.

**Comment by edwardlee-msft** [comment]
> 

This is a separate bug, but since it is somewhat related I have added it to this PR as well.
image

**Review by carocao-msft — APPROVED** [review]

**Review by dmceachernmsft — APPROVED** [review]

**Pull request merged**